### DISCUSSION TITLE
AqualinkD + Aqualink RS panel

### DISCUSSION THREAD

**Discussion by pevillarreal** [aqualinkd/AqualinkD#257] [Q&A]
Hi general dumb question here. 

If I wire the raspberry pi into the RS control panel inside the house can I only use the raspberry pi or the panel? IE. can they share the wires and both will work to control the pool equipment?

If not then I assume I have to wire in the raspberry pi in the main control panel outside if I want to control the equipment using both the RS Panel in the house and also via AqualinkD.

**Comment by ballle98** [comment]
You can have may things on the RS-485 connector on the control panel (Salt Water Generator, Variable speed pumps, PDA, and Remote SPA (RS) controller).  The PI is just another one of those things.  Basically you just have to set the device id of aqualinkd to an ID that is not being used by the existing equipment.  you may already have a RS multiplier in you panel with a connection that is not used.  You can add you pi there.  If not you can simply add the wires from the pi to the existing wires in parallel.  looking at your panel  you may already see multiple things combined into the RS-485 port.  Think of it the same way you would put a 3-way plug into an outlet.  I have probably 50 ft of cat 5e between my pi and the panel.  I ran the wire up the wall into the attic and then inside the house so the pi is inside the house.  You can probably run this 100ft before you start seeing issues.  Depending on your panel location you could just drill straight through the wall and put the pi inside the house.  I find that works better since a metal box will block most of the wifi signal.  Anyway hope this helps.

![image](https://github.com/sfeakes/AqualinkD/assets/2252524/f7d85f05-1c01-41b6-8fc1-3d057261f46d)

![image](https://github.com/sfeakes/AqualinkD/assets/2252524/8409cefe-f213-46ed-bd8d-1f32eb3a2e5e)
